end-to-end-untargeted-metabolomics.Rmd

```diff
@@ -824,45 +874,53 @@ Can also observe it in our ions of interest:
 #' Similar comparison but this time filtering to see cystine
 par(mfrow = c(1, 2), mar = c(4, 4.5, 1, 0.5))
 
-old_eic_cystine <- chromatogram(data_raw[QC_samples],
-                                mz = cystine_mz + c(-0.05, 0.05), 
+old_eic_cystine <- chromatogram(data_raw,
```

Comment:
I would show the results on the full data set, not just the QCs here. Also, these EICs will be reused in the correspondence analysis and with just the QC samples the code/plots there don't work.